pkg/cmd/roachtest/restore.go

```diff
@@ -306,17 +306,36 @@ func registerRestoreNodeShutdown(r *testRegistry) {
 }
 
 func registerRestore(r *testRegistry) {
+	largeVolumeSize := 2500 // the size in GB of disks in large volume configs
+
 	for _, item := range []struct {
-		nodes   int
+		nodes        int
+		cpus         int
+		largeVolumes bool
+
 		timeout time.Duration
+		tags    []string
 	}{
-		{10, 6 * time.Hour},
-		{32, 3 * time.Hour},
+		{nodes: 10, timeout: 6 * time.Hour},
+		{nodes: 32, timeout: 3 * time.Hour},
+		// Run a cluster with larger nodes weekly.
+		{nodes: 12, timeout: 3 * time.Hour, cpus: 16, largeVolumes: true, tags: []string{"weekly"}},
 	} {
+		clusterOpts := make([]createOption, 0)
+		testName := fmt.Sprintf("restore2TB/nodes=%d", item.nodes)
+		if item.cpus != 0 {
+			clusterOpts = append(clusterOpts, cpu(item.cpus))
+			testName += fmt.Sprintf("/cpus=%d", item.cpus)
+		}
+		if item.largeVolumes {
+			clusterOpts = append(clusterOpts, volumeSize(largeVolumeSize))
+			testName += "/largeVolume"
```

Comment:
I'd go ahead and name this /2.5tb-pd or something since "large" may change in the future

Reply:
Updated.